src/pyatsimagebuilder/utils.py

```diff
@@ -67,6 +68,39 @@ def ftp_retrieve(host, from_path, to_path, port=None, secure=False):
     ftp.close()
 
 
+def git_info(path, repo=None):
```

Comment:
not fully understand what this is for. maybe better to update README with this change and also doc relating to manifest?

Reply:
I've added the various discovery files with a brief description to the README.